config/routes.rb

```diff
@@ -1,8 +1,6 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: "shows#index"
-  resources :shows do
-    resources :users
-  end
-
+  resources :shows
+  resources :users
```

Comment:
I would ideally like to see these scoped down to only those routes you actually intend on utilizing.